prow/cmd/tackle/main.go

```diff
@@ -161,6 +231,48 @@ func selectProject(choice string) (string, error) {
 	return choice, nil
 }
 
+// selectZone returns the user-selected zone, defaulting to the current gcloud one.
+func selectZone() (string, error) {
+	const MaxZones = 20
+
+	def, err := currentZone()
+	if err != nil {
+		return "", fmt.Errorf("get current zone: %v", err)
+	}
+
+	fmt.Printf("Available zones:\n")
+
+	zoneList, err := zones()
+	if err != nil {
+		return "", fmt.Errorf("list zones: %v", err)
+	}
+
+	err = validateNotEmpty(zoneList)
```

Comment:
nit: The error values of `validateNotEmpty` and `validateContainment` aren't actually ever used and the caller will always be able to provide an error with more context so these funcs could just return bools instead of errors.